src/extension.ts

```diff
@@ -261,7 +260,7 @@ async function _activateExtension(
   // set up the local Docker event listener singleton and start watching for system events
   EventListener.getInstance().start();
   // reset the Docker credentials secret so `src/docker/configs.ts` can pull it fresh
-  getStorageManager().deleteSecret(SecretStorageKeys.DOCKER_CREDS_SECRET_KEY);
+  void context.secrets.delete(SecretStorageKeys.DOCKER_CREDS_SECRET_KEY);
```

Comment:
Starting to explicitly add `void` where we don't want to await/handle the promise here (as part of https://github.com/confluentinc/vscode/pull/1498)